docs/jwst/outlier_detection/outlier_detection.rst

```diff
@@ -23,20 +23,58 @@ Specifically, this routine performs the following operations:
     This allows each plane of the cube to be treated as a separate 2D image
     for resampling (if done) and for combining into a median image.
 
-* By default, resample all input images into grouped observation mosaics; for example,
-  combining all NIRCam multiple detector images from `a single exposure or
-  from a dithered set of exposures.
-  <https://jwst-docs.stsci.edu/near-infrared-camera/nircam-operations/nircam-dithers-and-mosaics>`_
-
+* By default, resample all input images.
+  - The resampling step starts by computing an output WCS that is large enough
+    to encompass all the input images.
+  - All images from the *same exposure* will get resampled onto this output
+    WCS to create a mosaic of all the chips for that exposure.  This product
+    is referred to as a "grouped mosaic" since it groups all the chips from
+    the same exposure into a single image.
+  - Each dither position will result in a separate grouped mosaic, so only
+    a single exposure ever contributes to each pixel in these mosaics.
+  - An explanation of how all NIRCam multiple detector group mosaics are
+    defined from `a single exposure or from a dithered set of exposures
+    <https://jwst-docs.stsci.edu/near-infrared-camera/nircam-operations/nircam-dithers-and-mosaics>`_
+    can be found here.
+  - The ``fillval`` parameter specifies what value to use in the ouptut
+    resampled image for any pixel which has no valid contribution from any
+    input exposure.  The default value of ``INDEF`` indicates that the value
+    from the last exposure will be used, while a value of 0 would result in
+    holes.
+  - The resampling can be controlled with the ``pixfrac``, ``kernel`` and
+    ``weight_type`` parameters.
+  - The ``pixfrac`` indicates the fraction by
```

Comment:
Unfortunately, the best description of these parameters in any STScI documentation can only be found in the Drizzlepac readthedocs pages.  The text already included here in the outlier_detection docs would be the second best description, while there are almost no useful docstrings or descriptions for any resampling parameters actually associated with the resample step itself.  I would add a link to the Drizzlepac readthedocs pages, except it would confuse those using JWST as to why they are suddenly directed to HST docs.